components/Feature/StatusForm/index.js

```diff
@@ -0,0 +1,85 @@
+import { REFERRAL_STATUSES } from 'lib/utils/constants';
+import { useState } from 'react';
+
+const StatusForm = ({ onSubmitForm }) => {
+  const [reject, setReject] = useState(false);
```

Comment:
I see this state being set to `false`. Is this reflected on the radio buttons? _(As in, the "No" radio is pre-selected)_

Reply:
Ah, this is to show/hide the optional comments textarea, so even if nothing is selected we want it to be false